### ISSUE TITLE
Lite updates on confirm.html

### ISSUE THREAD

**Issue by theclabs** [stellar-deprecated/sep24-demo-client#87]
In the next line there is a var in camelCase, the norm is snake-case. its show a "undefined" error in confirm page.

https://github.com/stellar/sep6-demo-client/blob/e9304206633c2a6fb7ab392e6f212311ad9098f2/src/steps/withdraw/poll_for_success.js#L31

I suggest some small improvements in the UI to show dynamic data. I have the fork with the mods and I plan to add all balances and assets to the main screen.

![image](https://user-images.githubusercontent.com/3954535/63752027-d8347480-c886-11e9-83f2-6590129685cb.png)

**Comment by msfeldstein** [comment]
Yep we should definitely make those dynamic.  You can do this by passing query parameters and use the [data-replace] attribute to replace those properties
